Define 'dashboard' and 'quickstart' (#9593)
newrelic/docs-website: src/content/docs/new-relic-solutions/get-started/glossary.mdx

    A Synthetic monitor feature that allows you to run Synthetic [monitors](#monitor) from within your own systems by creating [private minions](#minion). **Private locations** allow you to extend your Synthetic coverage to new geographical locations, and to monitor websites behind your firewall such as an intranet site. For more information, see [Private locations overview](/docs/synthetics/new-relic-synthetics/private-locations/private-locations-overview-monitor-internal-sites-add-new-locations).
  </Collapser>

  <Collapser
    id="quickstart"
    title="quickstart"
  >

> At New Relic, `quickstart` refers to methods of installation that can get you one or both of the following: 

Reviewer: ```suggestion
At New Relic, `quickstart` refers to all methods of installation needed to get you one or both of the following: 
```
